scripts/index.js

```diff
@@ -338,6 +338,12 @@
   };
 
   var setupPlayerStats = function(player) {
+    player.on('dispose', () => {
+      if (window.statsTimer) {
+        window.clearInterval(window.statsTimer);
+        window.statsTimer = null;
+      }
+    });
```

Comment:
~this does get cleared out when we make a new player.~
It was supposed to be cleared here but I accidentally put the lines backwards https://github.com/videojs/http-streaming/blob/e8230a9b22b9a1b3ccb8f0926798c5ef42f53ed6/scripts/index.js#L547-L551

Reply:
probably better to do it in the actual function on dispose anyway. Keep it all contained. I will remove those lines.

Reply:
sounds good